.env.production.example

```
@@ -60,7 +60,7 @@ JWT_SECRET=<64-char-hex-string>
 # Generate with: node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
 ENCRYPTION_KEY=<64-char-hex-string>
 # Shared secret: frontend uses this to call API endpoints that bypass Supabase JWT
-INTERNAL_API_KEY=<32+ char random string>
+INTERNAL_API_KEY=<generate-with-openssl-rand-hex-32>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The INTERNAL_API_KEY key should go before the JWT_SECRET key

(UnorderedKey)